pkg/client/client.go

```diff
@@ -47,6 +47,8 @@ import (
 
 	monv1 "github.com/coreos/prometheus-operator/pkg/apis/monitoring/v1"
 	monclientv1 "github.com/coreos/prometheus-operator/pkg/client/versioned/typed/monitoring/v1"
+
+	apibuildv1 "github.com/openshift/api/build/v1"
```

Comment:
because buildv1 is already imported on https://github.com/kiegroup/kogito-cloud-operator/blob/11e7f8df37b6d74ebda655b6df1120403b24df70/pkg/client/client.go#L45